app_dart/lib/src/service/stackdriver_logger.dart

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:googleapis/logging/v2.dart';
+import 'package:googleapis_auth/auth_io.dart';
+import 'package:http/http.dart';
+import 'package:meta/meta.dart';
+
+import '../datastore/cocoon_config.dart';
+
+/// Service class for interacting with the AppEngine project's stackdriver logging service.
+///
+/// Stackdriver is a Google Cloud service for monitoring and managing services.
+/// https://cloud.google.com/stackdriver/ for more details.
```

Comment:
Nit: make it a complete sentence: "See https://...".

Reply:
Good catch